pkgs/development/python-modules/country-list/default.nix

```
@@ -0,0 +1,34 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+
+  # build-system
+  poetry-core,
+}:
+
+buildPythonPackage rec {
+  pname = "country_list";
```

```suggestion
  pname = "country-list";
```
https://nixos.org/manual/nixpkgs/unstable/#contributing